scripts/openshiftci-presubmit-e2e-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Do not show token in CI log
+set +x
+
+# show commands
+set -x
+
+# Set necessary env variables
+export PATH="$PATH:$(pwd)"
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+# Install Argo CD
+make install-argocd-openshift
+
+# Deploy GitOps Service components
+# Use Image built by CI, image name is provided as env variable 'CI_IMAGE' by CI
+echo "Using Image $CI_IMAGE"
+make install-all-k8s IMG=$CI_IMAGE
+
+# Port forwarding is needed by E2E to connect with DB
+kubectl port-forward --namespace gitops svc/gitops-postgresql-staging 5432:5432 &
+KUBE_PID=$!
+
+# DB Password is required by E2E to connect with DB
+DB_PASS=$(kubectl get -n gitops secret gitops-postgresql-staging -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+make test-e2e DB_PASS=$DB_PASS
```

Comment:
Nit: We can also export the DB_PASS `variable` without passing it as an argument.

Reply:
We can but in that case it will print the password in logs, to avoid that we have to remove `set -x` but then we wont be able to see commands executed in logs which makes it easy to separate the logs for debugging.

![image](https://user-images.githubusercontent.com/63054462/232823043-5a2ba189-6dce-4f88-b4c9-76601209e89c.png)

Reply:
You can disable, then re-enable, logging, like:
```
set +x
# ( run a command you don't want to output )
set -x
```

Reply:
This is done.